chromium_src/chrome/browser/devtools/devtools_window.h

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_DEVTOOLS_DEVTOOLS_WINDOW_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_DEVTOOLS_DEVTOOLS_WINDOW_H_
+
+#define HatsNextWebDialogBrowserTest \
```

Comment:
minor: should be `Has` not `Hats`?

Reply:
That should be an upstream issue, we just use that string to add our friend class without additional patches